src/rgw/rgw_rest_s3.cc

```diff
@@ -2045,7 +2044,6 @@ void RGWPostObj_ObjStore_S3::send_response()
     s->formatter->dump_string("Key", s->object.name);
     s->formatter->close_section();
   }
-  s->err.message = err_msg;
```

Comment:
should either keep this line, or add `err_msg` as an argument to `set_req_state_err()` below